generators/auth/templates/mongodb/jwt/src/components/auth/interfaces/login-payload.interface.ts

```diff
@@ -1,10 +1,9 @@
 import { Types } from 'mongoose';
-import { RolesEnum } from '@decorators/roles.decorator';
 
 export interface LoginPayload {
   readonly id?: Types.ObjectId;
 
-  readonly email?: string;
+  readonly email: string;
 
-  readonly role?: RolesEnum;
+  readonly role: string;
```

Comment:
why role: string? Why have you replaced role with enum type by role with string type? I think with enum it's more clearer